swaggerConfig.js

```diff
@@ -26,6 +26,7 @@ const options = {
     servers: [
       {
         url: 'http://localhost:3030/',
+        //TODO: IN PROD "URL" MUST BE CHANGED
```

Comment:
Add a new object with url “https://race-fanatic-server.onrender.com/“

Reply:
Added new url and descriptions.